wp-parsely.php

```diff
@@ -47,12 +226,44 @@ function maybe_load_plugin() {
 	 *
 	 * add_filter( 'wpvip_parsely_load_mu', '__return_false' );
 	 */
-	if ( ! apply_filters( 'wpvip_parsely_load_mu', get_option( '_wpvip_parsely_mu' ) === '1' ) ) {
-		return;
-	}
+	$do_not_load_plugin = ! apply_filters( 'wpvip_parsely_load_mu', get_option( '_wpvip_parsely_mu' ) === '1' );
```

Comment:
having this spelled out and separate is useful

```suggestion
	$load_plugin_requested_via_option = get_option( '_wpvip_parsely_mu' ) === '1';
	$do_not_load_plugin = ! apply_filters( 'wpvip_parsely_load_mu', $load_plugin_requested_via_option );
```

Reply:
Awesome suggestion. Thanks 👍